src/events/order/cd-event-add-child-ticket.vue

```diff
@@ -76,6 +76,12 @@
         notes: '',
       };
     },
+    created() {
+      this.$validator.extend('overage', {
+        getMessage: field => `The ${field} value is invalid. Youth is overage (18+)`,
```

Comment:
> Overage
Even after learning that this is a valid spelling for this, I still can't read it as one. 😅 Can we go with "over age" or "over-age"..?

Reply:
yeah i will put the dash